packages/mwp-app-render/src/components/Dom.jsx

```diff
@@ -69,14 +69,22 @@ const DOM = props => {
 		escapedState,
 	};
 
+	// strip <script> html tags
+	// in order to dangerouslySetInnerHTML in JSX
+	const newrelicJS = newrelic
+		.getBrowserTimingHeader()
+		.replace(/<script[^>]+>/, '')
```

Comment:
I guess technically you could have `/<script[^>]*>/` here (`*` instead of `+`), which would allow it to work if they ever changed the implementation to start with `<script>` instead of `<script type='...'>`